scripts/bfasst/flows.py

```diff
@@ -61,14 +75,79 @@ def get_flow_fcn_by_name(flow_name):
     return fcn
 
 
-def run_flow(design, flow_type, build_dir, print_to_stdout=True):
+def run_flow(design, flow_type, flow_args, build_dir, print_to_stdout=True):
     assert type(design) is bfasst.design.Design
 
     flow_fcn = flows.get_flow_fcn_by_name(flow_type)
-    return flow_fcn(design, build_dir, print_to_stdout)
-
+    return flow_fcn(design, flow_args, build_dir, print_to_stdout)
+
+
+def parse_flow_args(flow_args):
```

Comment:
Sorry, I'm not quite sure what this function is doing.  It looks like it's manually parsing the arguments? I'm not quite sure what it's for...